cmd/docgen/run.go

```diff
@@ -91,10 +91,10 @@ func eventsForAction(actionJSON []byte) (json.RawMessage, error) {
 		return nil, err
 	}
 
-	events := session.Events()
-	eventJSON := make([]json.RawMessage, len(events))
-	for i, event := range events {
-		typed, err := utils.EnvelopeFromTyped(event)
+	eventLog := session.EventLog()
```

Comment:
Definitely open to naming discussion - we need terminology that makes it clear that these are not events but something else that contains an event and information about how it was generated